r53.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_route53_record" "domain_amazonses_verification_record" {
+  count   = "${var.zone_id != "" ? 1 : 0}"
+  zone_id = "${var.zone_id}"
+  name    = "_amazonses.${var.domain}"
+  type    = "TXT"
+  ttl     = "3600"
+  records = ["${aws_ses_domain_identity.domain.verification_token}"]
+}
+
+resource "aws_route53_record" "domain_amazonses_dkim_record" {
+  count   = "${length(var.dkim_records)}"
```

Comment:
You should also guard the `count` when no `zone_id` is given. Passing a map, but not a `zone_id` would fail here.

Reply:
fixed